practica2023/src/main/java/com/endava/practica2023/Controller/EventssController.java

```diff
@@ -0,0 +1,30 @@
+package com.endava.practica2023.Controller;
+
+import com.endava.practica2023.Domain.Eventss;
+import com.endava.practica2023.Domain.dto.EventssDTO;
+import com.endava.practica2023.Service.EventssService;
+import jdk.jfr.Event;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class EventssController {
+    private final EventssService eventssService;
+
+    public EventssController(EventssService eventssService){
+        this.eventssService = eventssService;
+    }
+
+
+    @GetMapping("/api/getEvents/{venueId}/{eventTypeName}")
+    public List<EventssDTO> getEventsByVenueAndName(
```

Comment:
@PathVariable needs to contain the name of the request parameter, otherwise if you rename venueId during a refactor, that renaming will affect as well the interface of the API.
So you should have something like:

@PathVariable("venueId") Long venueId,
@PathVariable("eventType") String eventType